packages/mint-components/src/index.html

```diff
@@ -11,6 +11,11 @@
     <script nomodule src="build/mint-components.js"></script>
   </head>
   <body>
-    <sqm-stencilbook></sqm-stencilbook>
+    <!-- <sqm-stencilbook></sqm-stencilbook> -->
+    <sqm-referral-table>
+      <sqm-referral-table-user-column></sqm-referral-table-user-column>
+      <sqm-referral-table-status-column></sqm-referral-table-status-column>
+      <sqm-referral-table-rewards-column></sqm-referral-table-rewards-column>
+    </sqm-referral-table>
```

Comment:
@Locrian24 Never commit this! Add a story with this HTML content instead.

```suggestion
    <sqm-stencilbook></sqm-stencilbook>
```